site/public/js/server.js

```diff
@@ -1615,3 +1615,26 @@ function enableKeyToClick(){
     });
   }
 }
+
```

Comment:
Is there a reason why this is specifically creating a form through JS ? 

It seems like this function should instead either use ajax or fetch?
I'm not sure if we agreed on transitioning fro ajax to the fetch webApi yet but you can instead try something like:

```js
fetch(buildCourseUrl(['flag_user_image']), {
  method: 'POST',
  headers: {
    'Content-Type': 'application/json',
  },
  body: JSON.stringify(  
     { 'user_id' : user_id, 'csrf_token' : csrf_token }
  ),
})
.then(response => response.json());
```
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch
or just use plain ajax

Reply:
I'm going to shelve this until a later PR.  See #5587.